### PR TITLE
Update summary field name to description in info objects

### PR DESCRIPTION
**Pull request by ChristopheDujarric**
This PR updates `summary` field names to `description` in info objects.

The OpenAPI specification indeed [specifies](https://spec.openapis.org/oas/v3.1.0#fixed-fields-0):

Field Name | Type | Description
-- | -- | --
summary | string | A short summary of the API.
description | string | A description of the API. CommonMark syntax MAY be used for rich text representation.

As multiple current summary fields include Markdown descriptions, this change makes it possible to nicely render the definition file in standard vendor tooling on the market. Example: https://bump.sh/christophedujarric/doc/openai

### CODE REVIEW

**Comment by benadamdev** [comment]
@ChristopheDujarric https://bump.sh/christophedujarric/doc/openai is nothing short of a piece of art. I wish all API references were as clean and beautiful this one.

**Comment by ChristopheDujarric** [comment]
Thanks @paulRbr !

File: openapi.yaml [hunk 1]
```diff
@@ -58,8 +58,9 @@ paths:
                             schema:
                                 $ref: "#/components/schemas/CreateChatCompletionResponse"
 
+            summary: Create chat completion
+
             x-oaiMeta:
-                name: Create chat completion
```
Comment (rattrayalex) [inline openapi.yaml:62]: Please don't remove x-oai things

Comment (paulRbr) [reply openapi.yaml:62]: cc @ChristopheDujarric https://github.com/ChristopheDujarric/openai-openapi/pull/2

Comment (ChristopheDujarric) [reply openapi.yaml:62]: thanks @rattrayalex and @paulRbr !

**Pull request closed without merging**